assets/i18n/en.json

```diff
@@ -32,9 +32,6 @@
     "I18N_ABOUT_PAGE_FOUNDATION_TAB_PARAGRAPH_3": "If you wish to join the hundreds of other individuals supporting these efforts, please learn more about <a href=\"/donate\">donating to the Oppia Foundation</a> or <a href=\"/contact\">getting involved in other ways</a>.",
     "I18N_ABOUT_PAGE_FOUNDATION_TAB_PARAGRAPH_4": "The directors of the Foundation are Ben Henning, Jacob Davis, and Sean Lip. The Foundation's <a href=\"https://drive.google.com/folderview?id=0B9KSjiibL_WDfjlpUUpqampZSTBOTmRqM2IyaGN3R3NlVWstZlhKcjlZQ3VMTHJBQWNGTFU&usp=sharing\" class=\"inline-links\">bylaws and minutes</a> are available to read. If you wish to contact the Foundation, please email: <a href=\"mailto:admin@oppia.org\" class=\"inline-links\">admin@oppia.org</a>.",
     "I18N_ABOUT_PAGE_FOUNDATION_TAB_PARAGRAPH_4_HEADING": "Directors",
-    "I18N_ABOUT_PAGE_FOUNDATION_TAB_PARAGRAPH_5": "All content in Oppia's explorations is licensed under <a href=\"https://creativecommons.org/licenses/by-sa/4.0/legalcode\" class=\"inline-links\">CC-BY-SA 4.0</a>.",
-    "I18N_ABOUT_PAGE_FOUNDATION_TAB_PARAGRAPH_5_LICENSE_HEADING": "License",
```

Comment:
Instead of removing this from all the places should we change the key to `I18N_LICENSE_PAGE_LICENSE_HEADING`?

Reply:
Ditto for other translations where we have only changed the keys*

Reply:
done